{{ cookiecutter.project_slug }}/requirements/base.txt

```diff
@@ -8,6 +8,13 @@ psycopg2-binary
 git+https://github.com/uisautomation/django-automationcommon@master#egg=django-automationcommon
 django-ucamwebauth>=1.4.5
 
+# API-related Django packages
+djangorestframework
+django-cors-headers
+drf-yasg
+django-crispy-forms
```

Comment:
Are we using this at all?

Reply:
As this is a cherry-pick of the preferences webapp PR, I suppose we should see if there was a subsequent commit in the preferences webapp which removed django-crispy-forms? AFAICT it's still in the current master.